### ISSUE TITLE
🐛 [User-reported Bug] nuisance_regression run: [On, Off] won't output minimal preprocessed timeseries

### ISSUE THREAD

**Issue by XinhuiLi** [FCP-INDI/C-PAC#1494]
**Describe the bug**

When we turn on and off nuisance regression at the same time, there is only nuisance regressed timeseries `space-template_desc-cleaned_bold` but not minimal preprocessed timeseries `space-template_desc-brain_bold` in the output directory. 

**To Reproduce**
1. Set the config:
```
nuisance_corrections: 

  2-nuisance_regression: 

    # this is a fork point
    #   run: [On, Off] - this will run both and fork the pipeline
    run: [On, Off]
```
2. Run C-PAC

**Expected behavior**

There should be both nuisance regressed timeseries `space-template_desc-cleaned_bold` and minimal preprocessed timeseries `space-template_desc-brain_bold` in the output directory. 

**Screenshots**

**Versions** 
 - C-PAC: 1.8.0
 - Container Platform: Docker 1.8.0

**Additional context**

**Comment by SucyLi** [comment]
Found something interesting - if we first set nuisance [On, Off] and ran, after the run finished, we then set [Off] to run again based on existing working directory. I expected to see the minimal preprocessed timeseries would be in the output directory but it was not there.

**Comment by shnizzedy** [comment]
Fixed in #1542
